docs/en/ingest-management/commands.asciidoc

```diff
@@ -64,6 +121,50 @@ verification.
 `--certificate-authorities <string>`::
 Comma-separated list of root certificates used for server verification.
 
+`--enrollment-token <string>`::
+Enrollment token to use to enroll {agent} into {fleet}. You can use
+the same enrollment token for multiple agents.
+
+`--fleet-server-cert <string>`::
+Certificate to use for exposed {fleet-server} HTTPS endpoint.
+
+`--fleet-server-cert-key <string>`::
+Private key to use for exposed {fleet-server} HTTPS endpoint.
+
+`--fleet-server-es <string>`::
+Start a {fleet-server} process when {agent} is started, and connect to the
+specified {es} URL. You can optionally pass a username and password in the
+URL, but it's recommended that you use a service token instead (see the
+`--fleet-server-service-token` flag).
+
+//QUESTION: Is this correct? ^^ The description in --help needs work (I've
+//opened an issue to get it fixed).
+
+`--fleet-server-es-ca <string>`::
+Path to certificate authority to use to communicate with {es}.
+
+`--fleet-server-host <string>`::
+{fleet-server} HTTP binding host (overrides the policy).
+
+`--fleet-server-insecure-http`::
+Expose {fleet-server} over HTTP. This option is not recommended because it's
+insecure. It's useful during development and testing, but should not be used in
+production. When using this option, you should bind {fleet-server} to the
+local host (this is the default).
+
+`--fleet-server-policy <string>`::
+Start a {fleet-server} process when {agent} is started, and use the
+specified policy.
+
+//QUESTION: Is this correct? The description in --help says, "Start and run a
```

Comment:
You can override the implied default that Fleet server would be set up with the 'Default Fleet Server Policy' by passing this in.  It is primarily for self-managed usage, when working with multiple variations of policy configuration for different Fleet Servers (or just to have the Fleet Servers individually modifiable if desired)